Chapters/1-1-Chapter1.md

```diff
@@ -0,0 +1,167 @@
+
+"Lovely morning, Liffa."
+
+Startled, I sat up to see a silhouette of Melna standing in the light, her hair dancing in time with the leaves.
+
+"How long have you been here?" I asked.
+
+"Just a moment. I figured you would be here today."
+
+I stayed silent, still listening for the leaves and my mother's voice.
+
+"You've been gone a while. Breakfast is ready soon."
+
+"I'll be there in a bit."
+
+She stared at me for a moment, I assume to size up my strange behavior.
+
+"There's a chill in the air," she remarked, "Don't stay out too long."
+
+---
+
+Kipnu was at the stove reheating a sweet mash of pohua with his morning face on. His hair resembled the furrowed nest of a bird that took a liking to the most crooked of branches. He looked up, locking his tired eyes to mine, and put on his laziest morning smile which only complimented his hair. I walked up to hug and nuzzle him good morning. Melna was squatting at a table in the great room, reading her daily missive of provincial news.
+
+"Anything new?", I asked her.
+
+"Not particularly. We're still running our typical surplus of fruits. People always request more of them than they actually need."
+
+Kipnu chortled, looking at me. "Liffa here could probably eat the entire surplus herself."
+
+"Sweets are my one joy in life," I replied with a smile. "Shame will never keep me from my one true love. Anyway, all the farmers I know dry-preserve what they don't sell."
+
+"I'll pretend I didn't hear that," said Melna.
+
+I stirred at the table, and then rose again to go bother Kipnu who had fallen asleep at the stove. With a gentle nudge, I moved him away and removed the pot from the focal point.
+
+"A leaf would be great," I told him as I fetched a spoon.
+
+Melna walked over to us to check on the food. "Let me make sure it's still like my mother used to make," she took out a spoon and dipped it into the pot. "Too thick - add some water"
+
+I pulled some water from the dew column at the center of the great room and returned to add it to the mix little by little, waiting for Melna's approval.
+
+Finally, we sat down to eat.
+
+A moment of quiet went by and I could see Melna's mind had moved onto a deeper thought. She released her pen and sat up in her chair, staring longingly out at the door. I could tell she was thinking about traveling again. The three of us had joined too early before we could roam across the world.
+
+"It's been too long," she said with a longing tone.
+
+Kipnu knew what she was thinking too. "Last time was when we went to the North coast to live with the Hunnin for a week," he said. "I'll never forget how cold that was. Who's idea was that again?" He never missed a chance to remind me.
+
+I defended my choice, "Hey, we all had fun uh... catching fish."
+
+Melna and Kipnu smiled, knowing what I really meant.
```

Comment:
Much nicer change, more subtle. It's almost as if they are sharing an inside joke.